docs/.vitepress/theme/utils/useSearchPlaceholder.ts

```diff
@@ -0,0 +1,184 @@
+import { ref, Ref, watch } from 'vue';
+
+const BRAND_STOPWORDS = {
```

Comment:
Noice.
We could consider moving this into a JSON file and reusing it in the workflow.